src/types/participantErrorTypes.ts

```diff
@@ -0,0 +1,7 @@
+export enum ParticipantErrorTypes {
+  CHAT_MODEL_OFF_TOPIC = 'Chat Model Off Topic',
+  INVALID_PROMPT = 'Invalid Prompt',
+  FILTERED = 'Filtered by Responsible AI Service',
+  OTHER = 'Other',
+  DOCS_CHATBOT_API = 'Docs Chatbot API Issue',
+}
```

Comment:
nit: I think I'd prefer for participant related types to be in the participant folder. Curious what other folks think here. Fine as is too.

Reply:
good point. we have some of that pattern already with some types in the `participant/constants.ts` file so I'll move it there.